scripts/setup-chain-validator.sh

```diff
@@ -1,19 +1,13 @@
-#!/usr/bin/env bash
-
-set -ex
+#!/bin/bash
```

Comment:
Please revert the shebang. You can't know where person has their bash shell and maybe they don't want you to use the /bin/bash one but rather one that they've defined.